### PR TITLE
Few fixes to recalculations snakemake call

### PR DESCRIPTION
**Pull request by pmb59**
This adds config that exists in the first snakemake call (reprocess) to the second (recalculations).

Tested with success.

### CODE REVIEW

File: Snakefile-sorting-hat [hunk 1]
```diff
@@ -396,6 +399,9 @@ rule produce_recalculations_call:
                 metadata_summary=../{input.metadata} \
                 bioentities_properties={params.bioentities_properties} \
                 skip_steps_file={params.skip_steps_file} \
+		atlas_exps={params.atlas_exps} \
+	       	tmp_dir={params.tmp_dir} \
+	      	atlas_meta_config={params.atlasmeta_config} \
```
Comment (anilthanki) [inline Snakefile-sorting-hat]: What if `atlas_meta_config` is `None`

Comment (anilthanki) [inline Snakefile-sorting-hat]: I don't see `atlas_meta_config` used in `Snakefile-recalculations`

Comment (pmb59) [reply Snakefile-sorting-hat]: good catch, I have removed it

Comment (pmb59) [reply Snakefile-sorting-hat]: I have added (for the reprocessing ) a WARNING case [99ba92d](https://github.com/ebi-gene-expression-group/bulk-recalculations/pull/58/commits/99ba92d2a92be1e35ea70a1d11796f52a04a17c6) and an explicit exit here [6d174d4](https://github.com/ebi-gene-expression-group/bulk-recalculations/pull/58/commits/6d174d454c20dd1f910938b2f6740d0b5c376d91)

**Review by anilthanki — APPROVED** [review]
LGTM

**Pull request merged**